src/test/java/Homework16.java

```diff
@@ -0,0 +1,8 @@
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+public class Homework16 extends BaseTest {
+    ChromeOptions options =new ChromeOptions();
+    options.addArguments("remote")

```

Comment:
Please kindly add the test here for homework 16.
Create a test case, registrationNavigation() using @Test annotation.